handbook/engineering/developer-insights/index.md

```diff
@@ -50,17 +50,32 @@ In addition to the [engineering principles and practices](../principles-and-prac
 
 ### Unblocking others is your highest priority
 
-If a teammate is blocked by you on a question, your approval, or a pull request review, your top priority is always to unblock them, either directly or through helping them find someone else who can, even if this takes time away from your own or your team's priorities.
+If a teammate is blocked by you on a question, your approval, or a pull request review, your top priority is always to unblock them, either directly or through helping them find someone else who can, even if this takes time away from your own or your team's priorities. If you're the one who is blocked, be sure to communicate that so that others can prioritize appropriately.
+
+Normally, waiting for a PR review does not mean you are blocked: it's expected that you can start working on something else (e.g. a new PR that depends on the first one). However, in some scenarios it's important to get a review ASAP — e.g., to fix a regression or a broken CI pipeline. In those cases, you should communicate the urgency and expect that your teammates will prioritize unblocking you.
```

Comment:
I'm a bit worried here about making it black-and-white blocked vs not blocked, when in reality a review delay is always at least an _inconvinience_ (because of uncertainty in  follow-up work, potential merge conflicts, needing to rebase stacked PRs, delay in dogfooding, ...). Maybe we can make it clear that this is not a wildcard to delay reviews for days or weeks after e.g. whatever project you're currently working on is done. Imo every dev should generally budget some time every day (e.g. in the morning) to do some PR reviews.
```suggestion
Normally, waiting for a PR review does not mean you are blocked: it's expected that you can start working on something else (e.g. a new PR that depends on the first one). However, in some scenarios it's important to get a review ASAP — e.g., to fix a regression or a broken CI pipeline. In those cases, you should communicate the urgency and expect that your teammates will prioritize unblocking you.
Even when a teammate is not _blocked_, but _inconvinienced_ (because of follow-up work), don't leave them hanging for extended periods of time. You should generally budget some amount of time every day for doing reviews.
```